src/py/flwr/common/context.py

```diff
@@ -48,7 +52,8 @@ def __init__(
         self,
         state: RecordSet,
         partition_id: Optional[int] = None,
+        run_config: Dict[str, str] = {},
```

Comment:
Collection type literals like `{}` and `[]` must never be used in default arguments:
```suggestion
        run_config: Dict[str, str],
```